### DISCUSSION TITLE
Turkish translation group

### DISCUSSION THREAD

**Discussion by emrecgty** [microsoft/ML-For-Beginners#196] [Translations]
This is the discussion place for Turkish translators to discuss progress, consistency of translations, and any encountered issues. 

See #153 for the progress of the translation.

**Comment by buseorak** [comment]
Hi! Any suggestions on how we can manage consistency in terminology?

I believe we can create a glossary along the way for quick lookup. :) I don't know what platform we can use to do that though...

**Reply by jlooper** [reply]
Hi, the French translators have created a great system you can emulate: https://github.com/microsoft/ML-For-Beginners/discussions/186

**Comment by buseorak** [comment]
While translating Classifiers-1 lesson, I realized two things. I would appreciate it if you could take a look at them:

- The CSV file under data folder is named as "cleaned_cuisine.csv" but in the readme file it is somethimes referred as "cleaned_cuisines.csv".

- In the 18th line of the Classifiers-1 readme file the CSV file path is entered as `pd.read_csv("../../data/cleaned_cuisine.csv")` although, as far as I observed, it could be `pd.read_csv("../data/cleaned_cuisine.csv")`

May I change them or leave them as they are?

**Reply by jlooper** [reply]
hi! This is one reason I love our community translators - you find all my mistakes! Yes, it should be named `cleaned_cuisines` as there are several cuisines listed. Let me do a tidy-up. Great catch!

**Reply by jlooper** [reply]
done!

**Reply by buseorak** [reply]
Thank you so much! :)
